.github/workflows/tests.yml

```
@@ -16,9 +16,9 @@ jobs:
     runs-on: ${{ matrix.config.os }}
     env:
       BOUT_TEST_TIMEOUT: "6m"
-      PETSC_DIR: /usr/lib/petscdir/3.7.7/x86_64-linux-gnu-real
+      PETSC_DIR: /usr/lib/petscdir/3.12.4/x86_64-linux-gnu-real
       PETSC_ARCH: ""
-      SLEPC_DIR: /usr/lib/slepcdir/3.7.4/x86_64-linux-gnu-real
+      SLEPC_DIR: /usr/lib/slepcdir/3.12.2/x86_64-linux-gnu-real
```

I think this needs to be:

```suggestion
      SLEPC_DIR: /usr/lib/slepcdir/slepc3.12/x86_64-linux-gnu-real/
```

Going off the [list of files in the package](https://packages.ubuntu.com/focal/amd64/libslepc-real3.12-dev/filelist)

---

Presumably there's a reason they did this, but it beats me